nsv13/code/datums/mood_events/nsv_events.dm

```diff
@@ -52,3 +52,7 @@
 	description = "<span class='nicegreen'>Cheers! ¡Salud! Kanpai! Prost! Skål! Santé! Sláinte! Saúde!</span>\n"
 	mood_change = 3
 	timeout = 30 SECONDS
+
+/datum/mood_event/lizard_vibrations
+	description = "<span class='warning'>I know my muscles are shaking to warm me up, but it still feels distracting..</span>\n"
+	mood_change = -2
```

Comment:
```suggestion

/datum/mood_event/lizard_vibrations
	description = "<span class='warning'>I'm shivering!</span>\n"
	mood_change = -2
```
I think it would be better to word this less technically, you could also consider calling it shivering instead of lizard vibrations, then, for a potential more general application

Reply:
I suppose this *is* essentially just shivering. Bit carried away on terminology. I do want to leave an indicator in the moodlet that this sort of shivering is because of the environmental cold.

Reply:
Adjusted. Still lizard specific since this is just three lines if someone wants to replicate it.